.github/workflows/test.yml

```diff
@@ -559,36 +558,37 @@ jobs:
             echo "Changelog config not applied" && exit 1
           fi
 
-          test-json:
-            runs-on: ubuntu-latest
-            steps:
-              - name: Checkout code
-                uses: actions/checkout@v3
-                with:
-                  path: "./"
-        
-              - run: npm ci --prod
-              - run: touch ./fake-file.log
-              - run: "git config --global user.email 'changelog@github.com'"
-              - run: "git config --global user.name 'Awesome Github action'"
-              - run: "git add . && git commit -m 'feat: Added fake file so version will be bumped'"
-        
-              - name: Generate changelog
-                id: changelog
-                uses: ./
-                env:
-                  ENV: 'dont-use-git'
-                  EXPECTED_TAG: 'v1.5.0'
-                with:
-                  github-token: ${{ secrets.github_token }}
-                  version-file: 'test-file.json'
-        
-              - name: Show file
-                run: |
-                  echo "$(<test-file.json)"
-        
-              - name: Test output
-                run: node ./test-output.js
-                env:
-                  FILES: 'test-file.json'
-                  EXPECTED_VERSION: '1.5.0'
+  test-conventionalcommits-preset:
```

Comment:
👍 Like that you're keeping the tests separate here.  Probably not very slow and will be more informative.

Reply:
These are all existing tests.  I simply added a new one following the existing conventions, but checking that we can load the `conventionalcommits` preset (which is currently the only one we actually use).